### DISCUSSION TITLE
Day Filters:  Only display meetings up to 12:00pm

### DISCUSSION THREAD

**Discussion by fbarnese** [code4recovery/12-step-meeting-list#1177] [Q&A] [answered]
Hi Team.

Harbor Area Central Office - www.hacoaa.org

When searching for meetings, filtering by day (not the default of Any Day), the database only returns meetings up through the noon hour (12pm).  The default of Any Time is on.  This is causing rumbles amongst the volunteer staff.

Is this a bug?  Or a setting that needs adjusting?

Please advise.

Faith Barnese
faith@barnesegroup.com

**Comment by joshreisner** [comment]
hi @fbarnese looks like a plugin called "ngo-charity-fundraising-pro" is adding a style to your site that is preventing it from knowing it is at the bottom of your page and to load more meetings. can you temporarily deactivate that plugin to confirm?

**Reply by joshreisner** [reply]
if that ends up being the issue we will try adding some CSS to TSML UI to defend against this problem since it came up once before in https://github.com/code4recovery/tsml-ui/issues/316

**Reply by joshreisner** [reply]
@fbarnese hm ok! looks like it resolved the scrolling issue but caused other issues - you can switch it back now!

we'll think of a solution with CSS.

**Reply by joshreisner** [reply]
@fbarnese could you temporarily add some custom CSS? i think this should be a band aid for now until we can roll out a fix. thanks

```
html { height: auto; }
```

**Reply by joshreisner** [reply, marked as answer]
@fbarnese ugh sorry i am confident this will work if you change it to 

```
html { height: auto !important; }
```

**Comment by fbarnese** [comment]
That's our theme Josh.  It has been working fine for the last 6 months.

However, I just switched to a different theme for you to check.

Faith
________________________________

**Comment by fbarnese** [comment]
Done.  Thanks Josh.
________________________________

**Comment by fbarnese** [comment]
It has been added but does not appear to have fixed the problem.
________________________________

**Comment by fbarnese** [comment]
It's a winner!  Thanks Josh.

Faith
________________________________
